### PR TITLE
chore: update changelog for version 2.11.8, 2.12.4, 2.13.2, and 2.14.4

### PR DESCRIPTION
**Pull request by erikayasuda**
- [x] update changelog for version 2.11.8, 2.12.4, 2.13.2, and 2.14.4

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,56 @@ Changelogs for versions not listed here can be found at https://github.com/DataD
...
+  - Resolves the issue where tracer flares would not be generated if unexpected types were received in the `AGENT_CONFIG` remote configuration product.
+
+---
+
+## 2.11.8
+
+
+### Bug Fixes
+
+- tracing(flare): Resolves the issue where tracer flares would not be generated if unexpected types were received in the AGENT_CONFIG remote configuration product.
```
Comment (vitor-de-araujo) [inline CHANGELOG.md:53]: This one doesn't have the same format as the others above :eyes:

Comment (erikayasuda) [reply CHANGELOG.md:53]: Ah, good catch, thanks!

**Review by vitor-de-araujo — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,57 @@ Changelogs for versions not listed here can be found at https://github.com/DataD
...
+
+---
+
+## 2.11.8
+
+
+### Bug Fixes
+
+- Tracing
+  - Resolves the issue where tracer flares would not be generated if unexpected types were received in the AGENT_CONFIG remote configuration product.
```
Comment (vitor-de-araujo) [inline CHANGELOG.md:54]:
```suggestion
  - Resolves the issue where tracer flares would not be generated if unexpected types were received in the `AGENT_CONFIG` remote configuration product.
```

**Pull request merged**